js/dataframe/index.tsx

```diff
@@ -476,6 +484,8 @@ export class ShinyDataFrameOutput extends HTMLElement {
 
     const myDiv = document.createElement("div");
     myDiv.classList.add("html-fill-container", "html-fill-item");
+    myDiv.style.marginLeft = "auto";
+    myDiv.style.marginRight = "auto";
```

Comment:
I don't love the centering. Do we do that anywhere else?

Reply:
See https://github.com/posit-dev/py-shiny/pull/1126#issuecomment-1961994819